lib/krane/cluster_resource_discovery.rb

```diff
@@ -101,6 +99,23 @@ def version_for_kind(versions, kind)
       version_override.fetch(kind, latest)
     end
 
+    def gvk_string(api_versions, resource)
+      apiversion = resource['apiversion'].to_s
+
+      ## In kubectl 1.20 APIGroups was replaced by APIVersions
+      if apiversion.empty?
+        apigroup = resource['apigroup'].to_s
+        group_versions = api_versions[apigroup]
+
+        version = version_for_kind(group_versions, resource['kind'])
+        apigroup = 'core' if apigroup.empty?
```

Comment:
cc/ @dturn I change this as well based on your comment https://github.com/Shopify/krane/pull/777#discussion_r552277061